repository/Seaside-WebSocket-Core.package/WAWebSocketPusher.class/instance/sockets.st

```diff
@@ -0,0 +1,5 @@
+accessing
+sockets
+	"Answer the current list of sockets."
+
+	^ sockets
```

Comment:
Isn't this error prone because the caller has to care about the mutex? Shouldn't we just offer `#socketsDo:`?